integrations/cloudbuild/chef.yaml

```diff
@@ -1,13 +1,13 @@
 steps:
-    - name: "ghcr.io/project-chip/chip-build-vscode:153"
+    - name: "ghcr.io/project-chip/chip-build-vscode:161"
```

Comment:
because of overlapping merges, we only got to build 162 version of vscode. 
https://github.com/project-chip/connectedhomeip/pkgs/container/chip-build-vscode

Could we update everything to 162?